pkg/plugins/optional/grafana/v1alpha/scaffolds/internal/templates/resources.go

```diff
@@ -34,7 +34,7 @@ func (f *ResourcesManifest) SetTemplateDefaults() error {
 	if f.Path == "" {
 		f.Path = filepath.Join("grafana", "controller-resources-metrics.json")
 	}
-
+	f.SetDelim("[[", "]]")
```

Comment:
Could you please add on top a comment explaining why are you seeing the Delim and why the values adopt are ("[[", "]]"? So that, it makes easier for others to understand it in the future if they need it.

Reply:
done